WordPress/Classes/Networking/BlogServiceRemoteREST.m

```diff
@@ -68,14 +66,14 @@ - (void)syncOptionsForBlog:(Blog *)blog
           }];
 }
 
-- (void)syncPostFormatsForBlog:(Blog *)blog
-                       success:(PostFormatsHandler)success
-                       failure:(void (^)(NSError *))failure
+- (void)syncPostFormatsForBlogID:(NSNumber *)blogID
+                         success:(PostFormatsHandler)success
+                         failure:(void (^)(NSError *))failure
 {
-    NSParameterAssert([blog isKindOfClass:[Blog class]]);
-    NSParameterAssert(blog.dotComID != nil);
+    NSParameterAssert([blogID isKindOfClass:[NSNumber class]]);
+    NSParameterAssert(blogID != nil);
```

Comment:
I don't think this assert is needed anymore. If `blogID == nil`, the previous assert will fail